.circleci/config.yml

```
@@ -140,6 +140,22 @@ jobs:
           command: bundle exec fastlane android build_magic_weather_compose
       - android/save-build-cache
 
+  build-custom-entitlement-computation:
```

Maybe
```suggestion
  build-custom-entitlement-computation-sample-app:
```
Since it's not building the SDK?